sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/functions.scala

```diff
@@ -842,3 +699,302 @@ object HyperLogLogPlusPlus {
   )
   // scalastyle:on
 }
+
+abstract class CentralMomentAgg(child: Expression) extends ImperativeAggregate with Serializable {
```

Comment:
Missing API doc. It is useful to cite the Wikipedia page. My version can be find here: http://arxiv.org/abs/1510.04923.